acme/acme/client.py

```diff
@@ -20,6 +19,8 @@
 from acme import errors
 from acme import jws
 from acme import messages
+# pylint: disable=unused-import, no-name-in-module
```

Comment:
Is this what you were talking about yesterday? When running lint with with Python 3.5+, the `no-name-in-module` error will go away correct?

Reply:
yes, this is what I was talking about.